fakelibusb_test.go

```diff
@@ -301,7 +321,9 @@ func newFakeLibusb() *fakeLibusb {
 		// The contents of these pointers is never accessed.
 		fd := new(fakeDevice)
 		*fd = d
-		fl.fakeDevices[newDevicePointer()] = fd
+		devPointer := newDevicePointer()
+		fl.fakeDevices[devPointer] = fd
+		fl.fakeLibUsbDevices = append(fl.fakeLibUsbDevices, devPointer)
```

Comment:
instead, add a new field to the fakeDevice type, and if fakeDevice.sysDevPtr is set, assign the value in the sys devices map. For testing I think it's fine to assume not all devices would be opened and available through a file descriptor.

Reply:
Right, got it.
The default of uintptr/int will be =0 though, should it be treated as a special value? This would be in case of the wrapSysDevice checking if a file descriptor/sysDevPtr is actually valid.